tools/ed64romconfig.c

```diff
@@ -51,9 +67,23 @@ int print_usage(const char * prog_name)
 	fprintf(stderr, "See: https://github.com/krikzz/ED64/blob/master/docs/rom_config_database.md#developer-override\n");
 	fprintf(stderr, "\n");
 	fprintf(stderr, "Command-line flags:\n");
-	fprintf(stderr, "\t-w, --savetype <type>  Declare cartridge save type.\n");
-	fprintf(stderr, "\t-c, --rtc              Declare real-time clock support.\n");
-	fprintf(stderr, "\t-r, --regionfree       Declare region-free ROM.\n");
+	fprintf(stderr, "\t-w, --savetype <type>           Declare cartridge save type.\n");
+	fprintf(stderr, "\t-c, --rtc                       Declare real-time clock support.\n");
+	fprintf(stderr, "\t-r, --regionfree                Declare region-free ROM.\n");
+	fprintf(stderr, "\t-1, --controller1 <type>        Define controller 1 hardware type. <type> should be one of:\n");
+	fprintf(stderr, "\t    default                     Emulators should follow standard config for this port.\n");
```

Comment:
Why would you ever explicitly set default instead of just omitting the parameter?